apps/network/tests/database/presets/role.py

```diff
@@ -0,0 +1,4 @@
+role_metrics = [
+    ("Administrator", False, False, False, False),
```

Comment:
You've already created these roles. While I'm sure this works, it might be a more clear test if you added a new type of user that won't already be seeded.